Makefile

```diff
@@ -2,18 +2,14 @@
 
 HTMLCOV_DIR ?= htmlcov
 
-test: flake8 test_lib
+test: flake8 pylint pytest
```

Comment:
They pick up different errors. Flake8 is pyflakes + pep8 checking, whereas pylint has its own set of rules. There is some overlap but it's nicer to use both.